src/test/tools/pmemobjcli/pmemobj_allocate.posc

```diff
@@ -0,0 +1,61 @@
+#!./pmemobjcli -s
```

Comment:
If I'm not mistaken you're using Michał's work in here ( https://github.com/micrec90/pmdk/commit/3caf7ba992d8ee24ee1c08ffdb8e9dfef0671234 ), then please close this PR and make it properly from the scratch, using his commit, not just mentioning him in "ref"

Reply:
Of course, that's why I have put annotation as you seen. Michal has put few fixes/improves in his branch, so firstly i tried to use his commit, but it was quite complicated and time consuming so I thought it will be either good to mention him in this PR, but if you wish to close, there is no problem.

Reply:
Unfortunately this is how it works, that's the way to not 'steal' his work.